paddle_billing/Notifications/Entities/Price.py

```diff
@@ -57,7 +57,3 @@ def from_dict(cls, data: dict) -> Price:
             created_at=datetime.fromisoformat(data["created_at"]) if data.get("created_at") else None,
             updated_at=datetime.fromisoformat(data["updated_at"]) if data.get("updated_at") else None,
         )
-
-
-# Prevents circular import
-from paddle_billing.Entities.Product import Product  # noqa E402
```

Comment:
`paddle_billing.Entities.Product` is not referenced in this file (can reinstate if it's still needed)